app/Services/ExchangeRateGetter.php

```diff
@@ -4,68 +4,93 @@
 class ExchangeRateGetter {
 	public function get() {
 		
-	/// Receive JSON exchange rates
+		/// RECEIVE JSON TODAY'S EXCHANGE RATES
 		$arrContextOptions = [
             "ssl" => [
                 "verify_peer" => false,
                 "verify_peer_name" => false,
             ],
         ];
 
-    $path = file_get_contents(
-    	"https://www.cbr-xml-daily.ru/daily_json.js", 
-    	false, 
-    	stream_context_create($arrContextOptions)
-    );
+	    $path = file_get_contents(
+	    	"https://www.cbr-xml-daily.ru/daily_json.js", 
+	    	false, 
+	    	stream_context_create($arrContextOptions)
```

Comment:
php 8 supports parameters (https://php.watch/versions/8.0/named-parameters)
Learn and use it